docs/core/app-types.md

```diff
@@ -34,8 +34,7 @@ There are two main types that we can observe:
 
 ## Portable applications
```

Comment:
I'm thinking we should find a different term than "portable" since these apps are not portable in the industry sense.